setup.cfg

```diff
@@ -2,7 +2,9 @@
 testpaths = tests
 
 [flake8]
-max-line-length = 120
+select = C, E, W, F, I
+max-line-length = 88
+extend-ignore = E203, W503
```

Comment:
Why E203 is ignored?

Reply:
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8